include/swift/AST/ASTScope.h

```diff
@@ -1945,17 +1945,55 @@ class CaseStmtScope final : public AbstractStmtScope {
 private:
   void expandAScopeThatDoesNotCreateANewInsertionPoint(ScopeCreator &);
 
+public:
+  std::string getClassName() const override;
+  Stmt *getStmt() const override { return stmt; }
+};
+
+class CaseLabelItemScope final : public ASTScopeImpl {
```

Comment:
Looks good. How about a comment for the two new scopes explaining what they model? Maybe a tiny Swift code example??

Reply:
Done.